.github/workflows/release.yaml

```diff
@@ -6,6 +6,8 @@ on:
       - main
       - test
       - dev
+    tags:
```

Comment:
I would suggest another workflow for this - the current `release` looks more a `deployment` workflow, not sure we want to mix triggers with this one